code/modules/organs/external/_external.dm

```diff
@@ -1023,3 +1017,12 @@ Note that amputating the affected organ does in fact remove the infection from t
 					flavor_text += "a ton of [wound]\s"
 		return english_list(flavor_text)
 
+/obj/item/organ/external/is_usable()
+	//return ..() && !is_stump() && !(status & ORGAN_TENDON_CUT) && (!can_feel_pain() || get_pain() < pain_disability_threshold) && brute_ratio < 1 && burn_ratio < 1
```

Comment:
Ah, yes, you probably should know about it since you are going to port brainmed. This is how this line is supposed to look with bay12's medicine. But i left it because i forgot to delete it.

Reply:
>still didn't delete it